packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -440,6 +513,9 @@ class _Table<T> extends StatefulWidget {
   final ColumnData<T> sortColumn;
   final SortDirection sortDirection;
   final Function(ColumnData<T> column, SortDirection direction) onSortChanged;
+  final FocusNode focusNode;
+  final void Function(RawKeyEvent event, ScrollController scrollController,
+      BoxConstraints constraints) handleKeyEvent;
```

Comment:
make a typedef for this

`typedef TableKeyEventHandler = void Function(RawKeyEvent event, ScrollController scrollController,
      BoxConstraints constraints);`

Reply:
done